src/main/java/mat/client/cqlworkspace/attributes/LightBoxRightPanelView.java

```diff
@@ -1,88 +1,44 @@
-                        "<select id=\"alergy_event_type\" name=\"alergy_event_type\">\n" +
-                        "<option value=\"First Choice\">First Choice</option>\n" +
-                        "<option value=\"Second Choice\">Second Choice</option>\n" +
-                        "<option value=\"Third Choice\">Third Choice</option>\n" +
-                        "<option value=\"Fourth Choice\">Fourth Choice</option>\n" +
-                        "</select>" +
-                        "</div>\n" +
-                        "\n" +
-                        "</div>"
-                        +
-                        "<div><h4>Assessment, Not Ordered</h4>\n" +
-                        "<div>\n" +
-                        "<h5><label for=\"assessment_event_code\">code</label></h5>\n" +
-                        "<select id=\"assessment_event_code\" name=\"assessment_event_code\">\n" +
-                        "<option value=\"First Choice\">First Choice</option>\n" +
-                        "<option value=\"Second Choice\">Second Choice</option>\n" +
-                        "<option value=\"Third Choice\">Third Choice</option>\n" +
-                        "<option value=\"Fourth Choice\">Fourth Choice</option>\n" +
-                        "</select>" +
-                        "</div>\n" +
-                        "\n" +
-                        "<div>\n" +
-                        "<h5><label for=\"assessment_event_type\">type</label></h5>\n" +
-                        "<select id=\"assessment_event_type\" name=\"assessment_event_type\">\n" +
-                        "<option value=\"First Choice\">First Choice</option>\n" +
-                        "<option value=\"Second Choice\">Second Choice</option>\n" +
-                        "<option value=\"Third Choice\">Third Choice</option>\n" +
-                        "<option value=\"Fourth Choice\">Fourth Choice</option>\n" +
-                        "</select>" +
-                        "</div>\n" +
-                        "\n" +
-                        "</div>"
-        );
-        html.setWidth("100%");
-        html.setHeight("100%");
+        for (FhirDataTypeModel dataTypeEditorModel : model.getDataTypesByResource().values()) {
+            FieldSet fs = new FieldSet();
+            fs.ensureDebugId("LightBoxRightPanelView_form_fieldset_" + dataTypeEditorModel.getId());
+            fs.add(new Legend(dataTypeEditorModel.getFhirResource()));
+            fs.setVisible(false);
+            for (FhirAttributeModel attr : dataTypeEditorModel.getAttributesByElement().values()) {
+                AttributeEditorDisplay attributeEditor = new AttributeEditorView(attr);
+                fs.add(attributeEditor.asWidget());
+                attributeEditors.put(attr.getId(), attributeEditor);
+            }
+            dataTypeEditors.put(dataTypeEditorModel.getId(), fs);
+            form.add(fs);
+        }
```

Comment:
I think lambdas make this a little cleaner but its just personal preference. Something like:

```
model.getDataTypesByResource().values().forEach(, m-> {
        FieldSet fs = new FieldSet();
        fs.ensureDebugId("LightBoxRightPanelView_form_fieldset_" + m.getId());
        fs.add(new Legend(m.getFhirResource()));
        fs.setVisible(false);
        m.getAttributesByElement().values().forEach(
                a -> attributeEditors.put(a.getId(), new AttributeEditorView(a)));
        dataTypeEditors.put(m.getId(), fs);
        dataTypeEditors.put(m.getId(), fs);
        form.add(fs);
     });
```